Ego4D-Future-Hand-Prediction/slowfast/datasets/ego4dhand.py

```diff
@@ -119,6 +120,8 @@ def _construct_loader(self):
                             if frame_type in ["action_start_sec", "action_end_sec","action_start_frame","action_end_frame","action_clip_start_sec","action_clip_end_sec","action_clip_start_frame","action_clip_end_frame"]:
                                 continue
                             # frame_gt = frame_annot[1]
+                            if len(frame_annot)==2:
```

Comment:
What's the error there?  Is it any set of annotations > 1?  (How many do you see offhand?)